### DISCUSSION TITLE
Error: The deploy step encountered an error: There was an error creating the worktree: The process '/usr/bin/git' failed with exit code 128 ❌ ❌

### DISCUSSION THREAD

**Discussion by alankarjain05** [JamesIves/github-pages-deploy-action#1364] [Q&A]
Getting the below error when deploying to gh-pages branch in remote. Any help to resolve this please?

![image](https://user-images.githubusercontent.com/39750116/236449787-cdf51191-7faa-4385-9ca0-1da79ff29843.png)

**Comment by archriste** [comment]
Have you tried adding
```
permissions:  
  contents: write
```
to your workflow file to rule out anything wrong with the `token` configuration?
